lib/styleDictionary/util.js

```diff
@@ -0,0 +1,98 @@
+const DEFAULT_KEY = 'default';
+
+/**
+ * Simple utility function to either wrap the given output object in an object
+ * as a prop with key 'propKey', or return an empty object. Usefull to create
+ * the desired CTI output of Style Dictionary.
+ * 
+ * @param String propKey: key in which the output should be wrapped 
+ * @param Object obj: object to wrap 
+ */
+const outputHelper = (propKey, obj) => {
+  const output = {};
+
+  if (Object.keys(obj).length) {
+    output[propKey] = obj;
+  }
+  
+  return output;
+}
+
+/**
+ * Transforms a dash-divided word to camelcase.
+ * Example: inner-shadow => innerShadow
+ *
+ * @param String str 
+ */
+const dashToCamelCase = str => `${str}`
```

Comment:
Please remove, we have [`camelcase`](https://github.com/sindresorhus/camelcase#readme) installed.